src/olympia/ratings/models.py

```diff
@@ -319,7 +319,7 @@ class RatingFlag(ModelBase):
     )
     FLAGS = (
         *USER_FLAGS,
-        (AUTO, _('Auto-flagged due to word match')),
+        (AUTO, _('Auto-flagged due to word match or user restriction')),
```

Comment:
Could/should we use a specific flag instead ?

Reply:
https://github.com/mozilla/addons-server/pull/20581/commits/431a7c1f1f95b3fcb10c0959db706b964ee88d02